projects/knora-ui/src/lib/viewer/values/link-value/link-value.component.spec.ts

```diff
@@ -194,16 +289,78 @@ describe('LinkValueComponent', () => {
 
     });
 
-    it('should return a selected resource', () => {
```

Comment:
why have you removed this test for the selected resource? It was there to check that the selected option is always an instance of ReadResuorce.